python/sglang/srt/layers/moe/fused_moe_triton/fused_moe.py

```diff
@@ -1537,7 +1538,7 @@ def fused_experts_impl(
                     gemm1_alpha,
                     gemm1_limit,
                 )
-            elif _is_cuda:
+            elif _is_cuda or _is_hip:
```

Comment:
There is another `gelu_and_mul` can be imported from `sgl_kernel `when `_is_hip ==True`.